tests/runner/Cargo.toml

```diff
@@ -1,16 +1,17 @@
 [package]
 name = "flv-test"
 version = "1.0.1"
-authors = ["fluvio.io"]
+authors = ["Fluvio Contributors <team@fluvio.io>"]
 edition = "2018"
+autotests = false
 
 [dependencies]
 log = "0.4.8"
-bytes = "0.5.3"
```

Comment:
don't update this yet since other parts of fluvio is not upgrade to this version